### DISCUSSION TITLE
Could not query post data Prismic examlpe

### DISCUSSION THREAD

**Discussion by AlexanderVishnevsky** [vercel/next.js#12116] [Help] [answered]
Hi, there. I decided to try brand new example with integration of [Prismic CMS](https://github.com/zeit/next.js/tree/canary/examples/cms-prismic). Main page looks good, but when I try to open some post it gives  error 404. Debugger shows only empty objects. It's probably because of broken query.
 
![image](https://user-images.githubusercontent.com/26820006/80022198-5ca5c880-84e4-11ea-8181-23a02d9c579a.png)

**Comment by lfades** [comment]
404 means the post you created wasn't found, for troubleshooting try the following:

- Make sure that you have the correct API key and project URL set in `.env`
- Make sure you have created at least one post, and that it's published, every time you create or edit a post, it goes into draft mode after saving it, you have to publish it manually
- Check that the post appears in the list of posts (`/`)

If everything is okay, please provide more details that can help reproduce the issue.

**Reply by AlexanderVishnevsky** [reply]
root page works well, you can see working query 

![image](https://user-images.githubusercontent.com/26820006/80026651-24ee4f00-84eb-11ea-8344-70746ad2ad45.png)

But post page gives error
![image](https://user-images.githubusercontent.com/26820006/80026830-73035280-84eb-11ea-9c22-a8431e501ae5.png)

You can try it by yourself: https://retro-cams.prismic.io/graphql
```
query PostBySlug($slug: String!) {
  post(uid: $slug, lang: "en-us") {
    title
  }
}

_Query variables:_
{"slug": "qwerty"}

```

**Comment by AlexanderVishnevsky** [comment, marked as answer]
I found the bug 🐛. If you create your Prismic account before and change your default locale from 'en-us' to another, this example won't work. I made [PR ](https://github.com/zeit/next.js/pull/12132)with configuration of locale.

It can be closed right now.
